dist/tools/openocd/openocd.sh

```diff
@@ -136,6 +152,29 @@ do_flash() {
     echo 'Done flashing'
 }
 
+do_flash_elf() {
+    test_config
+    test_elffile
+    test_elffile_antibrick
```

Comment:
I believe this is a little too specific and does not belong in `openocd.sh`. Give it a more generic name and rename the environment variable `ANTIBRICK` accordingly.